docs/getting-started/index.rst

```diff
@@ -11,85 +11,163 @@ Getting started
+    :title: How to compute a 2D image
+    :link: ../tutorials/index.html#d-image
 
-    +++
+Gammapy treats 2D maps as 3D cubes with one bin in energy. Sometimes, you might want to use previously
+obtained images lacking an energy axis (eg: reduced using traditional IACT tools) for modeling and fitting
+inside Gammapy. In this case, it is necessary to attach an `energy` axis on as it is showm in the tutorials.
 
-    .. link-button:: upgrading
-            :type: ref
-            :text: To the upgrade
-            :classes: btn-block btn-secondary stretched-link
+.. accordion-footer::
 
-    ---
-    :column: col-12 p-3
 
-    Installation by other methods
-    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. accordion-header::
+    :id: collapseEight
+    :title: How to compute a lightcurve
+    :link: ../tutorials/analysis/time/light_curve.html
 
-    Installing with pip or other package managers.
+The light curve estimation tutorial shows how to extract a run-wise lightcurve.
 
-    +++
+To perform an analysis in a time range smaller than that of an observation, it
+is necessary to filter the latter with its `~gammapy.data.Observations.select_time` method. This produces
+an new observation containing events in the specified time range. With the new
+`~gammapy.data.Observations` it is then possible to perform the usual data
+reduction which will produce datasets in the correct time range. The light curve
+extraction can then be performed as usual with the
+`~gammapy.estimators.LightCurveEstimator`. This is demonstrated in the light curve flare tutorial.
 
-    .. link-button:: other-install
-        :type: ref
-        :text: Learn more
-        :classes: btn-secondary stretched-link
\ No newline at end of file
+.. accordion-footer::
+
+.. accordion-header::
+    :id: collapseTwelve
+    :title: How to detect sources in an image
+    :link: ../tutorials/analysis/2D/detect.html
+
+Gammapy provides methods to perform source detection in a 2D map. First step is
+to produce a significance map, i.e. a map giving the probability that the flux
+measured at each position is a background fluctuation. For a
+`~gammapy.datasets.MapDataset`, the class `~gammapy.estimators.TSMapEstimator` can be
+used. A simple correlated Li & Ma significance can be used, in particular for
+ON-OFF datasets. The second step consists in applying a peak finer algorithm,
+such as `~gammapy.estimators.utils.find_peaks`.
+
+.. accordion-footer::
+
+.. accordion-header::
+    :id: collapseTwenty
+    :title: How to combine data from multiple instruments
+    :link: ../tutorials/analysis/3D/analysis_mwl.html
+
+Gammapy offers the possibility to combine data from multiple instruments
+in a "joint-likelihood" fit.
+
+.. accordion-footer::
+
+
+.. _download-tutorials:
+
+Download Tutorial Datasets
+--------------------------
+
+You can now proceed to download the Gammapy tutorial notebooks and the example
+datasets. The total size to download is ~180 MB. Select the location where you
```

Comment:
Maybe we can also mention that the catalogs are also located here. Sometimes people got the idea that the catalogs are accessed directly from the "websites", whereas GAMMAPY_DATA is only for the DL3 data

Reply:
I agree this would be good to explain, however I think the "Getting Started" page is not the right place...it should not give any details but just help people to get started. I'll think about a better place...